examples/helm-provider/test-helm.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.crossplane.io/v1beta1
+kind: Release
+metadata:
+  name: hello-example
+spec:
+  forProvider:
+    chart:
+      name: hello
+      repository: https://cloudecho.github.io/charts
```

Comment:
is there another chart we can use? I have no idea what cloudecho is. Maybe [this](https://helm.sh/docs/intro/quickstart/#install-an-example-chart) which is referenced by the official [eks doc](https://docs.aws.amazon.com/eks/latest/userguide/helm.html)

Reply:
Great idea! Will update.